frontend/catalyst/compiler.py

```diff
@@ -76,6 +78,7 @@ class CompileOptions:
     lower_to_llvm: Optional[bool] = True
     static_argnums: Optional[Union[int, Iterable[int]]] = None
     abstracted_axes: Optional[Union[Iterable[Iterable[str]], Dict[int, str]]] = None
+    multi_threaded_compilation: bool = False
```

Comment:
```suggestion
    multi_threaded_compilation: bool = True
```

shouldn't it be enabled by default, or is there an issue if we enable it by default?